src/app/components/AccountMenu/index.tsx

```diff
@@ -68,14 +68,16 @@ function AccountMenu({ title, balances, showOptions = true }: Props) {
           {title || <Skeleton />}
         </p>
 
-        {balances.satsBalance && balances.fiatBalance ? (
+        {balances.satsBalance ? (
           <p className="flex justify-between">
             <span className="text-xs dark:text-white">
               {balances.satsBalance}
             </span>
-            <span className="text-xs text-gray-600 dark:text-neutral-400">
-              ~{balances.fiatBalance}
-            </span>
+            {!!balances.fiatBalance && (
```

Comment:
Wondering if when this is active and the API takes a bit longer, will the loading-skeleton is already gone but the fiat value might still be i.e. `~0$`?

Reply:
Can you explain again what this Skeleton is good for? So far I did not experience any loading issues.

Reply:
Not blocking. We can have a look once this is merged.